### PR TITLE
Add property to limit output size of a query

### PR DESCRIPTION
**Pull request by mayankgarg1990**
Test plan -

1) Added unit tests
2) Manually tested by using the TpchQueryRunner:

```
$ cat /tmp/query.sql 
SET SESSION query_max_output_bytes='1000B';
SELECT * FROM orders;

$ ./presto-cli/target/presto-cli-0.241-SNAPSHOT-executable.jar --catalog tpch --schema sf30000 -f /tmp/query.sql > /tmp/out
SET SESSION
Query 20200902_000014_00001_9ki9x failed: Query has exceeded output size Limit of 1000B
```

```
== RELEASE NOTES ==

General Changes
* Add session property ``query_max_output_bytes`` and configuration property ``query.max-output-bytes`` to control how much data a query can output.
```

### CODE REVIEW

**Comment by fgwang7w** [comment]
LGTM. @mayankgarg1990 the logic here is very similiar to enforceScanLimits and looks good

**Review by tdcmeehan — APPROVED** [review]
LGTM

File: presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java [hunk 1]
```diff
@@ -403,6 +404,18 @@ public QueryManagerConfig setQueryMaxScanRawInputBytes(DataSize queryMaxRawInput
         return this;
     }
 
+    public DataSize getQueryMaxOutputSize()
```
Comment (tdcmeehan) [inline presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java:407]: Actually, add a `@MinDataSize("1B")` here?

**Comment by mayankgarg1990** [comment]
@tdcmeehan - all tests have successfully finished - we should be able to merge this PR now.

**Pull request merged**